### DISCUSSION TITLE
Renkler ve işlemleri sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#198] [Zeka Sorusu]
Mavi . Kırmızı = 28
Mor . Yeşil = 15
Siyah - Yeşil = 0

Yukarıdaki bağıntılara göre;

Mavi . Sarı - Turuncu = ?

**Comment by akgembi12** [comment]
Her rengim harf sayısı kadar değeri vardır
4.4-7=9

**Comment by tolga0102** [comment]
28 = 4.7
15 = 3.5
0 = 5-5
Bakınca buradaki her kelimenin harf sayısına göre bir sayıyı temsil ettiğini görüyoruz buna göre
4 . 4 - 7 = 9 olur.
